### PR TITLE
Make the ACL update message less noisy

### PR DESCRIPTION
**Pull request by turt2live**
Fixes https://github.com/vector-im/element-web/issues/14216

Copy is not reviewed as of writing.

Instead of saying "TravisR has changed the server ACLs for this room: example.org is now banned", we can just leave it at "TravisR changed the server ACLs for this room."

### CODE REVIEW

**Comment by t3chguy** [comment]
Given that we're removing i18n strings, can we do our seldom run prunei18n :P?

**Comment by turt2live** [comment]
We're not removing any i18n strings. We're actually *adding* them here.

**Comment by t3chguy** [comment]
Oh dear. I blindly thought the deleted changeset had i18n support :'(

**Comment by turt2live** [comment]
So did I when I ran `yarn i18n`. Was sad to see no results :(

**Review by jryans — APPROVED** [review]

**Comment by jryans** [comment]
@turt2live Personally, this seems uncontroversial to me, so perhaps Design review is not needed...? This text is basically the same, just with server names removed.

**Comment by turt2live** [comment]
I'm happy to use that rationale - design might want to look at it still, but I suspect they would want to look at it regardless of these changes.

File: src/TextForEvent.js [hunk 1]
```diff
@@ -198,59 +198,30 @@ function textForRelatedGroupsEvent(ev) {
...
-    if (unallowedServers.length > 0) {
-        changes.push(`Servers matching ${unallowedServers.join(", ")} were removed from the allowed list.`);
-    }
-
-    if (prev.allow_ip_literals !== current.allow_ip_literals) {
-        const allowban = current.allow_ip_literals ? "allowed" : "banned";
-        changes.push(`Participating from a server using an IP literal hostname is now ${allowban}.`);
+    // If we know for sure everyone is banned, mark the room as obliterated
+    if (current.allow.length === 0) {
+        return text + " " + _t("🎉 All servers are banned from participating! This room can no longer be used.");
```
Comment (Half-Shot) [inline src/TextForEvent.js:221]: Glad the sarcastic :tada: lives on

**Pull request merged**